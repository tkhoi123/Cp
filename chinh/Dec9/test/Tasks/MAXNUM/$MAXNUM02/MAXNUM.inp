8
29 29 292 584 29 112 338 679 