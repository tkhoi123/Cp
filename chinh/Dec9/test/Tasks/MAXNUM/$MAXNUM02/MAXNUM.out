679584338292929292112